chrome-launcher/chrome-launcher.ts

```diff
@@ -171,6 +166,11 @@ export class Launcher {
       this.chromePath = installations[0];
     }
 
+    // Kill spawned Chrome process in case of ctrl-C.
+    if (defaults(this.opts.handleSIGINT, true)) {
+      process.on(_SIGINT, this.sigintListener.bind(this));
```

Comment:
Good catch about bind returning a new object. Not sure I follow what/where you are proposing with keeping and saving the arrow function?